src/mozilla_sec_eia/cli.py

```diff
@@ -35,9 +37,19 @@ def formatter(prog) -> argparse.HelpFormatter:
     subparsers = parser.add_subparsers(required=True)
 
     # Add command to validate filing archive contents
-    validate_parser = subparsers.add_parser("validate")
+    validate_parser = subparsers.add_parser("validate_archive")
     validate_parser.set_defaults(func=lambda: GCSArchive().validate_archive())
 
+    # Add command to fine-tune ex21 extractor
+    validate_parser = subparsers.add_parser("finetune_ex21")
+    validate_parser.add_argument("--model-output-dir", default="layoutlm_trainer")
+    validate_parser.add_argument("--test-size", default=0.2)
+    validate_parser.set_defaults(func=train_model)
+
+    # Add command to rename labeled filings on GCS
+    validate_parser = subparsers.add_parser("rename_filings")
```

Comment:
Is there any reason to keep a validation parser (`validate_parser`) and make a separate parser for handling the labeled data (`label_parser`), or can you just make one generic parser (`parser`) that handles all the parsing?

Reply:
This is using a pattern documented [here](https://docs.python.org/3/library/argparse.html#sub-commands), which is a bit weird, but allows you to create sub-commands with their own unique arguments.

So by using multiple sub-parsers like this, we can run multiple commands with their own options all from the same top-level command like this:

`mozilla validate_archive {validate archive specific options}`
`mozilla finetune_ex21 {fine tune specific options}`

Reply:
Oh got it. I guess I'm asking if `rename_filings` and `validate_archive` + `finetune_ex21` should all have use the same parser, or if there should be a different parser for `rename_filings`